docs/v0.2.0-beta.7/quickstart.md

```diff
@@ -51,11 +51,11 @@ import { Model } from 'ember-cli-mirage';
 export default Model;
 ```
 
-The model will create an `authors` table in Mirage's *in-memory database*. The database enables our mocks to be dynamic, and lets us change the return data without rewriting the entire mock. In this way, we can share a single set of mock routes in both development and testing, while still having control over the response data.
+The model will create an `authors` table in Mirage's *in-memory database*. The database makes our route handlers dynamic&mdash;we can change the returned data without rewriting the handler. In this way, we can share a single implementation for a route in both development and testing, while still having control over the response data.
```

Comment:
how about

"In this way, we can share the same route definitions in both development and testing, while still having control over their response data."